xds/src/main/java/io/grpc/xds/internal/security/certprovider/CertificateProvider.java

```diff
@@ -45,6 +46,8 @@ public interface Watcher {
 
     void updateTrustedRoots(List<X509Certificate> trustedRoots);
 
+    void updateSpiffeRoots(Map<String, List<X509Certificate>> spiffeRoots);
```

Comment:
nit: Here and throughout, we should use the language of "SpiffeTrustMap" instead of "SpiffeRoots".

Rationale: Unlike the "trusted roots" case (where there is no data structure), we have to remember the data structure that the SPIFFE trust map comes with.

Reply:
Friendly ping, this was not addressed. :)